cc_library/py-scc/src/sciler/scclib/app.py

```diff
@@ -247,7 +246,9 @@ def __check_message(self, contents):
                 self.device.reset()
                 logging.info(("instruction performed", instruction))
             else:
-                (success, failed_action) = self.device.perform_instruction(action)
+                (success, failed_action) = self.device.perform_instruction(
+                    action
+                )  # TODO: remove failed_action as return argument
```

Comment:
is this something to do in this MR?

Reply:
I want to keep the javascript version and the python as much a like as possible

Reply:
I made an issue for this GH-149